rpc/src/module/subscription.rs

```diff
@@ -126,9 +127,13 @@ impl SubscriptionRpc for SubscriptionRpcImpl {
 }
 
 impl SubscriptionRpcImpl {
+    // remove `allow` tag when https://github.com/crossbeam-rs/crossbeam/issues/404 is solved
+    #[allow(clippy::zero_ptr, clippy::drop_copy)]
     pub fn new<S: ToString>(notify_controller: NotifyController, thread_name: Option<S>) -> Self {
         let new_block_receiver =
             notify_controller.subscribe_new_block(thread_name.as_ref().unwrap().to_string());
+        let new_transaction_receiver =
+            notify_controller.subscribe_new_transaction(thread_name.as_ref().unwrap().to_string());
```

Comment:
An out-of-scope suggestion: the parameter `thread_name` must be `Some(_)` because the `unwrap` here. Hence `thread_name: Option<S>` can be changed to `thread_name: S`.

Reply:
updated, please review again, thanks.